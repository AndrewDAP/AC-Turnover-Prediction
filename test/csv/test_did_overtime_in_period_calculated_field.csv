EMPLOYEE_ID,PERIOD_START,WORK_HOURS_DEVIATION_PER_PERIOD
104.0,2018-08-27,0.0
104.0,2018-08-28,0.5
