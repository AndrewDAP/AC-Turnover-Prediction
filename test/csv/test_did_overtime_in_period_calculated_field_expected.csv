EMPLOYEE_ID,PERIOD_START,WORK_HOURS_DEVIATION_PER_PERIOD,DID_OVERTIME_IN_PERIOD
104.0,2018-08-27,0.0,0
104.0,2018-08-28,0.5,1
